### PR TITLE
CLOUDSTACK-8756:Incorrect guest os mapping in CCP 4.2.1-6 for CentOS 5.9

### PR DESCRIPTION
**Pull request by shwetaag**
Check the bug 8756 for more details

### CODE REVIEW

**Comment by remibergsma** [comment]
@shwetaag Please add more details on the actual problem (also in Jira) and what you want to achieve by adding this test. Both CentOS 5 and CCP 4.2.1seem very specific (and old) to me. But I might be missing something so let me know :-)

**Comment by shwetaag** [comment]
@remibergsma   the issue was of incorrect mapping of centos guest os (grater than 5.9 ) in cloud stack .CS was sending wrong info to Vcenter to launch VMs and centos VMs guest os is set to others .
This will cause CentOS( 5.9 or any other greater version)  unable to be provided on VMware certain operations like attach and detach iso.
so this test case is about verifying that attach and detach iso is working fine with vmware VMs  especially when using centos guest os

**Comment by shwetaag** [comment]
result information :

@desc:Incorrect guest os mapping in vmware for CentOS 5.9 and above ... === TestName: test2_attach_ISO_in_CentOSVM | Status : SUCCESS ===
ok

----------------------------------------------------------------------
Ran 1 test in 392.003s

OK

**Comment by remibergsma** [comment]
@shwetaag I have no VMware so cannot run this test. Is there any way to make the test more generic?

File: tools/marvin/marvin/config/test_data.py [hunk 1]
```diff
@@ -775,6 +775,15 @@
         "ostype": "CentOS 5.6 (64-bit)"
 
     },
+    "CentOS6.3template": {
+        "displaytext": "Centos",
+        "name": "Centos",
+        "passwordenabled": False,
+        "ostype": "CentOS 6.3 (64-bit)",
+        "url": "http://10.147.28.7/templates/centos63.ova",
```
Comment (bhaisaab) [inline tools/marvin/marvin/config/test_data.py]: Can you share or host this ova publicly? Will this test fail for this URL/file not available in one's env.

Comment (remibergsma) [inline tools/marvin/marvin/config/test_data.py]: Agree, and this should be parameterised IMHO.

Comment (shwetaag) [inline tools/marvin/marvin/config/test_data.py]: @bhaisaab this is a simple centos template for vmware so nothing special about it. for this test to pass in one's environment he should have centos template uploaded in their http server and change the url accordingly . 
If you have a place where we can share this file publicly i will be more than happy to upload it there. Not only this file, i will upload  lot other  files  whose urls are a private http server mentioned in test_data.py

**Comment by sanju1010** [comment]
LGTM

**Comment by nitt10prashant** [comment]
LGTM

**Comment by karuturi** [comment]
Hi @sanju1010 please use git-pr script to merge pull requests. Its in the source code at tools/git/git-pr
Here is a doc on how to use it https://cwiki.apache.org/confluence/pages/viewpage.action?pageId=61311655
If you face any issues, please ping me or @remibergsma.  Do not do direct commits like above to master. only merge commits should be in master.

**Comment by karuturi** [comment]
I reverted the directed commit and merged this pr.

**Comment by miguelaferreira** [comment]
Thanks @karuturi ! :+1:

**Pull request closed without merging**